controllers/sms-gateway.js

```diff
@@ -8,9 +8,10 @@ var async = require('async'),
     recordUtils = require('./record-utils'),
     STATUS_MAP = {
       UNSENT: 'received-by-gateway',
+      PENDING: 'forwarded-by-gateway',
```

Comment:
"PENDING" is the gateway state and "forwarded-by-gateway" is the webapp state. PENDING is correct just in a different context. This enum is used to map between the two contexts.